pkg/commands/loaders/branches.go

```diff
@@ -87,7 +88,7 @@ outer:
 		if err != nil {
 			return nil, err
 		}
-		branches = append([]*models.Branch{{Name: currentBranchName, DisplayName: currentBranchDisplayName, Head: true, Recency: "  *"}}, branches...)
+		branches = slices.Prepend(branches, &models.Branch{Name: currentBranchName, DisplayName: currentBranchDisplayName, Head: true, Recency: "  *"})
```

Comment:
Prepend, Remove, Delete, and Insert are very handy and far more natural than the mental gymnastics that using `append` requires.

The only downside to Prepend is that if you pass a slice to prepend, it's not immediately obvious that the items won't be prepended in reverse. But that's only a small thing.